src/tests/Feature/Admin/LogTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Tests\Feature\Admin;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\File;
+
+class LogTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testLaravelLogDownloadSuccess()
+    {
+        if (!File::exists(storage_path('logs'))) {
```

Comment:
You should avoid mutating anything real in a test. In this case, that's the filesystem. See https://phpunit.de/manual/6.5/en/test-doubles.html#test-doubles.mocking-the-filesystem for information on how to mock the filesystem.

Reply:
I had tried to use `Storage::fake()` and that didnt work and the other lib they suggested looks like its not maintained anymore so didnt wanna add it as a dep.

Reply:
I don't feel comfortable approving a test that mutates the filesystem.

Reply:
I commented out the offending two tests for now and added a todo with note